Tasks/VsTestPlatformToolInstallerV1/task.json

```diff
@@ -10,12 +10,12 @@
         "Agent",
         "DeploymentGroup"
     ],
-    "minimumAgentVersion": "2.103.0",
+    "minimumAgentVersion": "2.144.0",
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 151,
-        "Patch": 3
+        "Minor": 211,
```

Comment:
Could you bump version according to the current sprint number

Reply:
Hello,
I have updated the version according to the current sprint, please review it.

Reply:
In task.json file, the version still shows as 211, Can you update it to 212?

Reply:
Typing get deprecated, using @type is the suggested way to install dependencies, I have followed the doc for upgrading azure pipeline task to node 10.

Reply:
updated the version in task.json file.